oak_core/src/sync.rs

```diff
@@ -40,6 +40,14 @@ impl<T> OnceCell<T> {
         }
     }
 
+    /// Gets an unwrapped reference to the inner value.
+    ///
+    /// Will panic if the cell has not been initialized.
+    pub fn get_unwrapped(&self) -> &T {
+        self.get().expect("cell not initialized")
+    }
```

Comment:
Not sure what the value of this is, may as well unwrap / expect manually at the calling site. Or does it mirror some existing API?

Reply:
I just thought it would be simler to do it in one place, but I agree that calling expect at the call site makes it clearer.